### PR TITLE
Search

### PR DESCRIPTION
**Pull request by maddie-jones**
#### Pivotal URL or Waffle Card number:
https://www.pivotaltracker.com/n/projects/2319720
#### What does this PR do?
Adds call to Musixmatch API 
Prints all songs from the searched artist with name and genre.
Adds favorite button to each song

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,2 +1,58 @@
-// This file is in the entry point in your webpack config.
 import './styles.scss'
+const artistInput = document.getElementById("search-input")
+const findSongsBtn = document.getElementById("find-song-btn")
+const apiKey = "869c7b19121345facdc061b2aa12dabf"
+var $ = require("jquery");
```
Comment (stoic-plus) [inline lib/index.js:5]: if we have time - I would recommend looking into ENV variables so api_key is not just hanging out

File: lib/index.js [hunk 2]
```diff
@@ -1,2 +1,58 @@
...
+   var t = document.createTextNode(`Genre: ${veriGenre}`)
+   par.appendChild(t)
+   document.getElementById("song").appendChild(par);
+
+   var button= document.createElement("BUTTON")
+   var t = document.createTextNode("Favorite")
+   button.appendChild(t)
+   document.getElementById("song").appendChild(button);
+  }
+}
```
Comment (stoic-plus) [inline lib/index.js:56]: this function could use a refactor - you could create HTML as a string and then append a div whose innerHTML is that string. I can show you tomorrow. 

Moving the genre to a separate function is good though.

**Review by stoic-plus — APPROVED** [review]

File: lib/index.js (hunk 2 above)
Comment (maddie-jones) [reply lib/index.js:56]: look forward to learning that way of doing it!

File: lib/index.js (hunk 1 above)
Comment (maddie-jones) [reply lib/index.js:5]: I will get on this right away.

**Pull request merged**